tests/integration/customizations/s3/test_plugin.py

```diff
@@ -575,7 +575,7 @@ def test_copy_metadata_directive(self):
         response = self.head_object(bucket_name, new_key)
         # These values should have the metadata of the source object
         metadata_ref = copy.copy(metadata)
-        metadata_ref['Expires'] = 'Thu, 01 Jan 1970 00:00:00 GMT'
+        metadata_ref['Expires'] = '1970-01-01T00:00:00+00:00'
```

Comment:
Good point. Looks like the integration test is picking up the actual value I had in my config file which is bad.